src/Eto/Forms/Controls/Control.cs

```diff
@@ -131,7 +131,9 @@ protected virtual void OnSizeChanged(EventArgs e)
 	public const string KeyDownEvent = "Control.KeyDown";
 
 	/// <summary>
-	/// Occurs when a key has been pressed and is down
+	/// Occurs when a key has been pressed and is down.
+	/// Focus is required for this event to be raised. Ensure CanFocus is set to true.
```

Comment:
CanFocus is only on a Drawable.  Other controls intrinsically can get focus like TextBox, TextArea, DropDown, CheckBox, etc when they are Enabled